content/en/docs/ops/app-health-check/index.md

```diff
@@ -121,7 +121,7 @@ The configuration changes above (by Helm or by the configuration map) effect all
 
 <!-- Add samples YAML or kubectl patch? -->
 
-Rather than install Istio with different Helm option, you can annotate Pod with `sidecar.istio.io/rewriteAppHTTPProbers: "true"`.
+Rather than install Istio with different Helm option, you can annotate Pod with `sidecar.istio.io/rewriteAppHTTPProbers: "true"` under `spec.template.metadata.annotations`.  If you annotate under `metadata.annotations`, the annotation is ignored.
```

Comment:
This clarification and warning seems inappropriate here. It's a general comment about how to add pod annotations to a deployment. Maybe there's some Kubernetes doc that this could point to: `you can [annotate the pod](link) with ...`

Reply:
k8s has docs for what is k8s objects (which includes pod), and annotate k8s objects so I tried to link those.